### PR TITLE
management command to delete documents from couch

### PR DESCRIPTION
**Pull request by czue**

### CODE REVIEW

File: corehq/apps/cleanup/management/commands/purge_docs.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        database = Domain.get_db()
+        for domain in domain_names:
+            for doc_type in doc_types:
+                docs = [row['doc'] for row in database.view(
+                    'domain/docs',
+                    startkey=[domain, doc_type],
+                    endkey=[domain, doc_type, {}],
+                    reduce=False,
+                    include_docs=True,
+                )][:remaining]
```
Comment (dannyroberts) [inline corehq/apps/cleanup/management/commands/purge_docs.py]: Ha! It never occurred to me that `[:]`, etc. is short for `[None:None]`

Comment (czue) [inline corehq/apps/cleanup/management/commands/purge_docs.py]: ha! you know i didn't explicitly think about that but it worked in testing and i didn't think twice. this is arguably too clever, but i think i'm ok with that

**Comment by czue** [comment]
@dannyroberts fyi no need to wait for tests since this is just a mgmt command

**Comment by dannyroberts** [comment]
word

**Comment by czue** [comment]
@gcapalbo I'm planning on using this to delete `XFormsSessions` from couch. any reason I shouldn't do that?

**Comment by gcapalbo** [comment]
@czue nope, no reason - fine to delete them from couch

**Comment by czue** [comment]
all set

**Comment by gcapalbo** [comment]
@czue nice, thanks for taking care of this

**Pull request merged**